web/src/lightbox.js

```diff
@@ -155,7 +155,7 @@ export class PanZoomControl {
     zoomIn() {
         const w = $(".image-preview").width();
         const h = $(".image-preview").height();
-        this.panzoom.smoothZoom(w / 2, h / 2, 1.5);
+        this.panzoom.smoothZoom(w / 2, h / 2, 2);
```

Comment:
Trying this out ... 2x is really too big of a zoom step; I think `sqrt(2)` would likely be considerably better...

Reply:
Updated to √2.